crates/holochain/src/conductor/conductor/tests/add_agent_infos.rs

```diff
@@ -2,6 +2,10 @@ use crate::sweettest::{SweetConductor, SweetDnaFile};
 use holochain_wasm_test_utils::TestWasm;
 
 #[tokio::test(flavor = "multi_thread")]
+#[cfg_attr(
+    target_os = "windows",
+    ignore = "get agent infos for cell returns None for unknown reason"
```

Comment:
Might use a consistent ignore string for these windows ones so its easy to find them all later.

Reply:
done https://github.com/holochain/holochain/pull/4839/commits/5cbab09e236035e0c58b80f072da2b05af4ef88c